### PR TITLE
Turn on TF Logging

### PR DESCRIPTION
**Pull request by rylew1**
## Summary
Fixes #{ISSUE}

## Changes proposed
 Add `TF_LOG` to the build and publish workflow step thats failing `Configure AWS credentials`

### CODE REVIEW

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -51,6 +51,8 @@ jobs:
 
       - name: Configure AWS credentials
         uses: ./.github/actions/configure-aws-credentials
+        env:
+            TF_LOG: DEBUG
```
Comment (rylew1) [inline .github/workflows/build-and-publish.yml:55]: not sure if this is the right place or if it should be in the `configure-aws-credentials/action.yml` file instead

Comment (coilysiren) [reply .github/workflows/build-and-publish.yml:55]: Try putting it in that file instead

Comment (rylew1) [reply .github/workflows/build-and-publish.yml]: Updated @coilysiren

**Review by coilysiren — APPROVED** [review]
Lets try it 👍🏼

**Pull request merged**